src/main/scala/uk/gov/hmrc/scalatestaccessibilitylinter/gateways/Vnu.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.scalatestaccessibilitylinter.gateways
+
+import nu.validator.client.EmbeddedValidator
+
+import java.io.ByteArrayInputStream
+
+object Vnu {
+  private lazy val vnu               = new EmbeddedValidator()
+  def embedded(html: String): String = vnu.validate(new ByteArrayInputStream(html.getBytes("UTF-8")))
```

Comment:
Same with above comment, can this just be made an implementation detail of the default embedded VNU linter? We could do away with the gateways package entirely.